auth-layer-proxy/configs/envoy-auth-template.yaml

```diff
@@ -32,18 +32,24 @@ static_resources:
                     name: ":method"
                     exact_match: "POST"
                 route:
-                  cluster: local_service
+                  cluster: upstream_service
 
   clusters:
-  - name: local_service
+  - name: upstream_service
     connect_timeout: 5s
-    type: LOGICAL_DNS
+    type: ${SERVICE_TYPE}
     load_assignment:
-      cluster_name: local_service
+      cluster_name: upstream_service
       endpoints:
       - lb_endpoints:
         - endpoint:
             address:
               socket_address:
-                address: host.docker.internal
-                port_value: 8020
+                address: ${SERVICE_ADDRESS}
+                port_value: ${SERVICE_PORT}
+
+admin:
```

Comment:
If possible order the yaml sections alphabetically, so `admin` would be up top somewhere

Reply:
✅  Done! 👍